src/test/java/com/google/sps/api/CalendarClientHelperTest.java

```diff
@@ -0,0 +1,103 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.api;
+
+import com.google.sps.servlets.SampleServlet;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mockito;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import com.google.api.services.calendar.model.Event;
+import com.google.api.services.calendar.model.EventAttendee;
+import javax.management.Attribute;
+
+@RunWith(JUnit4.class)
+public final class CalendarClientHelperTest {
+
+  private static final String ACCEPTED = "accepted";
+  private static final String DECLINED = "declined";
+  private final CalendarClientHelper calendarClientHelper = new CalendarClientHelper();
+  private static EventAttendee ATTENDING_SELF = new EventAttendee();
+  private static EventAttendee DECLINED_SELF = new EventAttendee();
+  private static EventAttendee NOT_RESPONDED_SELF = new EventAttendee();
+  private static EventAttendee ATTENDING_GUEST = new EventAttendee();
+  private static EventAttendee DECLINED_GUEST = new EventAttendee();
+  private static EventAttendee NOT_RESPONDED_GUEST = new EventAttendee();
```

Comment:
I wouldn't make these static because you want to make sure you get new instances every time a test is run to avoid potential side effects.

Reply:
Can you hint on how to create such resources before every test?

Reply:
https://github.com/junit-team/junit4/wiki/Test-fixtures
has a good overview of the available `@BeforeClass`, `@BeforeEach` and similar annotations you can use in tests to setup state that is needed in every test.

Reply:
Ohh, great, thanks!